webrtc.html

```diff
@@ -6486,9 +6486,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               transceiver, as defined in <span data-jsep="stopped">
               [[!JSEP]]</span>, the user agent MUST run the
               above steps as if <code>stop</code> had been called.
-              However in addition, the 
-              <code><var>transceiver</var>.receiver.track.onended</code>
-              Eventhandler is fired.</p>
+              In addition, the <code>ended</code> event is fired on
+              <code><var>transceiver</var>.receiver.track</code>.</p>
```

Comment:
Is there a way this could point to https://www.w3.org/TR/mediacapture-streams/#track-ended, to the steps that begin:

> When a MediaStreamTrack track ends for any reason other than the stop() method being invoked...